.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "kibana-proxy"]
 	path = kibana-proxy
 	url = https://github.com/fabric8io/openshift-auth-proxy.git
+[submodule "lib/openshift-ansible"]
```

Comment:
Using a submodule for a versioned, vendored dependency seems reasonable. Doing a submodule versus a clone and checkout is a matter of semantics, right? They achieve the same thing. Some people do not like the UX of submodules so it may just boil down to preference ... my opinion would be that it is better to store a dep using a submodule than using a specific checkout in some script somewhere.